Source/WebCore/platform/network/cf/DNSResolveQueueCFNet.cpp

```diff
@@ -39,6 +39,7 @@
 #include <wtf/StdLibExtras.h>
 #include <wtf/URL.h>
 #include <wtf/cf/VectorCF.h>
+#include <wtf/posix/SocketPosix.h>
```

Comment:
```suggestion
#include <wtf/posix/SocketPOSIX.h>
```
Consider leaving POSIX all capitals.